### ISSUE TITLE
fix(authN): fix AUTH_TYPE warning log in tests.

### ISSUE THREAD

**Issue by michalkrzyz** [cloudoperators/heureka#271]
**Task Description**
Fix AUTH_TYPE warning in tests
Either add default config AUTH_TYPE 'none' or remove warning log completely from the code.

**Acceptance Criteria:**
- [ ] No warning logs about AUTH_TYPE should be displayed during the tests

**Expected Test:**
Run tests and check log output.

**Comment by lolaapenna** [comment]
This problem is most likely no longer relevant and dependent on #239

**Comment by lolaapenna** [comment]
Pending @drochow  review.
